runtime/doc/lua.txt

```diff
@@ -1258,20 +1258,24 @@ vim.bo                                                                *vim.bo*
     Get or set buffer-scoped |local-options|. Invalid key is an error.
 
     This is a wrapper around |nvim_buf_set_option()| and
-    |nvim_buf_get_option()|.
+    |nvim_buf_get_option()|.  By default, sets option for the current buffer;
+    to set the option for the buffer with number `bufnr`, use `vim.bo[bufnr]`.
```

Comment:
This isn't even true anymore. It's now a wrapper of `nvim_get_option_value`. Same for win.

Reply:
Good catch; perfect time to fix that.